onboard/catkin_ws/src/controls/launch/controls.launch

```diff
@@ -11,7 +11,7 @@
         <arg name="sim" value="$(arg sim)" />
     </include>
 
-    <!--node pkg="tf2_ros" type="static_transform_publisher" name="base_link_publisher" args="0 0 0 0 0 0 1 odom base_link" /-->
+    <node pkg="tf2_ros" type="static_transform_publisher" name="base_link_publisher" args="0 0 0 0 0 0 1 odom base_link" />
```

Comment:
Not sure what this is about, might be something we tried when struggling with debugging?

Reply:
Don't merge